system/Helpers/kint_helper.php

```diff
@@ -16,7 +16,9 @@
         /**
          * Prints a Kint debug report and exits.
          *
-         * @param array ...$vars
+         * @param mixed $vars
```

Comment:
```suggestion
         * @param array ...$vars
```

Dont use mixed, we want specify type. Applies to all

Reply:
See #6310